.travis.yml

```
@@ -7,11 +7,25 @@ python:
 before_install:
   - python3 -m pip install --upgrade pip setuptools wheel
 install:
-  - pip3 install git+https://github.com/nextstrain/cli
+  # https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html#the-travis-yml-file
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - source "$HOME/miniconda/etc/profile.d/conda.sh"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  # Useful for debugging any issues with conda
+  - conda info -a
+  # Install nextstrain cli
+  - conda install -n base -c conda-forge mamba --yes
+  - conda activate base
+  - mamba create -n nextstrain -c bioconda nextstrain-cli --yes
+  - conda activate nextstrain
+  - mamba install -c conda-forge -c bioconda augur auspice nextalign snakemake git --yes
```

With nextstrain/docs.nextstrain.org#102 the recommendation is to have just one `mamba create` vs. `mamba create` + `mamba install`:

```suggestion
  - mamba create -n nextstrain -c bioconda nextstrain-cli augur auspice nextalign snakemake git --yes
  - conda activate nextstrain
```